Formula/p/pypy.rb

```diff
@@ -81,6 +82,17 @@ def install
       s.gsub! "/include'", "/include/tcl-tk'"
     end
 
+    if OS.mac?
+      # Allow python modules to use ctypes.find_library to find homebrew's stuff
+      # even if homebrew is not a /usr/local/lib. Try this with:
+      # `brew install enchant && pip install pyenchant`
+      inreplace "lib_python/2.7/ctypes/macholib/dyld.py" do |f|
+        f.gsub! "DEFAULT_LIBRARY_FALLBACK = [",
+                "DEFAULT_LIBRARY_FALLBACK = [ '#{HOMEBREW_PREFIX}/lib', '#{Formula["openssl@3"].opt_lib}',"
```

Comment:
```suggestion
                "DEFAULT_LIBRARY_FALLBACK = [ '#{HOMEBREW_PREFIX}/lib',"
```
OpenSSL used to be keg-only, which is why we needed to add it there separately. But it's no longer keg-only so we don't need to do that anymore.